docs/docs/installation.md

```diff
@@ -101,7 +101,7 @@ If you use [Create React App](https://github.com/facebookincubator/create-react-
 
 #### Webpack
 
-Include both `DefinePlugin` and `UglifyJsPlugin` into your production Webpack configuration as described in [this guide](https://webpack.js.org/guides/production-build/).
+Include both [`DefinePlugin`](https://webpack.github.io/docs/list-of-plugins.html#defineplugin) and [`UglifyJsPlugin`](https://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin) into your production Webpack configuration.
```

Comment:
I couldn't find an equivalent guide on production builds on the `1.x` site, so I just linked to the docs for each plugin instead.